src/structure_new/src/model_evaluator/4C_structure_new_model_evaluator_multiscale.cpp

```diff
@@ -97,6 +106,52 @@ void Solid::ModelEvaluator::Multiscale::determine_optional_quantity()
     }
   }
 }
```

Comment:
The method `determine_optional_quantity()` is no longer necessary with runtime output. It was just used temporarily to call prepare_output() which you now introduced properly.